flake-info/README.md

```diff
@@ -181,26 +181,23 @@ Enter the [nixos/nix](https://hub.docker.com/u/nixos/) docker image and proceed
 
 Note that you also need to have nix flakes support.
 
-Once you have nix installed run the following commands:
+Once you have nix (2.4 or later) installed run the following commands:
 
-1. ```
-   $ nix-shell -I nixpkgs=channel:nixos-21.05 -p nixFlakes
-   ```
-   to enter a shell with the preview version of nix flakes installed.
-2. ```
-   $ mkdir -p ~/.config/nix
-   $ echo "experimental-features = nix-command flakes" > .config/nix/nix.conf
-   ```
-   to enable flake support
+```
+$ mkdir -p ~/.config/nix
+$ echo "experimental-features = nix-command flakes" > .config/nix/nix.conf
+```
+
+to enable flake support.
 
 ### Installation, finally
 
 This project is defined as a flake therefore you can build the tool using
 
 ```
-$ nix build <project root>
+$ nix build <project root>#flake-info
 or
-$ nix build github:miszkur/github-search
+$ nix build github:nixos/nixos-search#flake-info
```

Comment:
Should we have a separate PR to update the readme?

Reply:
Why split?